bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -72,6 +73,39 @@ type Tuple []TupleElement
 // an instance of this type.
 type UUID [16]byte
 
+// Versionstamp is struct for a FoundationDB verionstamp. Versionstamps are
+// 12 bytes long composed of a 10 byte transaction version and a 2 byte user
+// version. The transaction version is filled in at commit time and the user
+// version is provided by your layer during a transaction.
```

Comment:
```suggestion
// version is provided by the application to provide an ordering of results within a transaction.
```

Something like that?